packages/reputation-miner/ReputationMiner.js

```diff
@@ -489,17 +496,56 @@ class ReputationMiner {
   }
 
   /**
-   * Submit what the client believes should be the next reputation state root hash to the `ReputationMiningCycle` contract
+   * Get the active reputation mining cycle
    * @return {Promise}
    */
-  async submitRootHash() {
+  async getActiveRepCycle() {
     const addr = await this.colonyNetwork.getReputationMiningCycle(true);
-    const repCycle = new ethers.Contract(addr, this.repCycleContractDef.abi, this.realWallet);
+    return new ethers.Contract(addr, this.repCycleContractDef.abi, this.realWallet);
+  }
+
+  /**
+   * Submit what the client believes should be the next reputation state root hash to the `ReputationMiningCycle` contract
+   * @param startIndex What index to start searching at when looking for a valid submission
+   * @return {Promise}
+   */
+  async submitRootHash(startIndex = 1) {
```

Comment:
The intention here is for this parameter to be passed with the index after any index they have already submitted to the current cycle, in order to avoid attempting to resubmit a hash with an index they have already used.

Upon reflection, this isn't strictly optimal behaviour for the miner - it skips any earlier entries that might now be valid that were not valid earlier. I think that behaviour change would be more appropriate in the 'autonomous mining' work I'm intending to do as part of #211.